oauth2-authorization-server/src/test/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2AuthorizationServerConfigurerTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.config.annotation.web.configurers.oauth2.server.authorization;
+
+import org.junit.Test;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class OAuth2AuthorizationServerConfigurerTest {
```

Comment:
These tests should be in the integration tests instead of a new test class.

However, I also feel adding these tests to the integration tests is a bit too much.

On second thought, let's remove these tests as I feel it's not necessary to test these scenarios out. Apologies for that.

Reply:
@jgrandja updated, please review again, thx!